test/fixtures/hello-cloud/template-runner.yml

```diff
@@ -0,0 +1,23 @@
+apiVersion: v1
+kind: PodTemplate
+metadata:
+  name: hello-cloud-template-runner
+  labels:
+    name: hello-cloud-template-runner
+    app: hello-cloud
+template:
+  metadata:
+    name: task-runner
+    labels:
+      name: runner
+      app: hello-cloud
+  spec:
+    containers:
+    - name: task-runner
+      image: hello-world:latest
```

Comment:
A template is added to the relevant project. For example Shopify would include a template specifying what a shopify pod looks like. This infrastructure can then use the template to actually run tasks.